cdap-ui/app/features/workflows/controllers/tabs/runs/tabs/status-ctrl.js

```diff
@@ -108,93 +106,6 @@ angular.module(PKG.name + '.feature.workflows')
 
   });
 
-/**
- * Adds start and end nodes to nodes list.
- * @param {Array} of nodes.
- */
-function addStartAndEndNodes(nodes, edges) {
-  if (nodes.length) {
-    nodes.unshift({
-      name: 'start',
-      type: 'START',
-      nodeType: 'START'
-    });
-    edges.unshift({
-      sourceName: nodes[0].name,
-      sourceType: nodes[0].nodeType,
-      targetName: nodes[1].name
-    });
-
-    nodes.push({
-      name: 'end',
-      type: 'END',
-      nodeType: 'END'
-    });
-    edges.push({
-      sourceName: nodes[nodes.length - 2].name,
-      sourceType: nodes[nodes.length - 2].nodeType,
-      targetName: nodes[nodes.length - 1].name
-    });
-
-  }
-
-}
-
-/**
```

Comment:
This comment belongs to the function below it.. I don't believe we should remove this one?